### PR TITLE
Remove UI + Controller logic for some Substrate Modules

### PR DESCRIPTION
**Pull request by zakhap**
## Description

Removes UI/Controller support for:
- Bounties
- Councils
- Collectives
- Phragmen Candidacies
- Identity Judgements

Does **not** touch `chain-events` package

## Does this PR affect any server routes?
chose YES or NO

## Have you added the issue number here?
(In the right sidebar, under "development")

### CODE REVIEW

**Comment by zakhap** [comment]
Screen Shot 2023-02-06 at 3 51 34 PM

accidentally ended up here from the sidebar in `/edgeware`

File: packages/commonwealth/client/scripts/controllers/chain/substrate/democracy_referendum.ts [hunk 1]
```diff
@@ -332,11 +327,6 @@ export class SubstrateDemocracyReferendum extends Proposal<
       });
     if (democracyProposal) return democracyProposal;
 
-    const collectiveProposal = chain.council?.store.getAll().find((p) => {
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/controllers/chain/substrate/democracy_referendum.ts:335]: A little worried about this -- what happens in the case where it _is_ a collective proposal?

File: packages/commonwealth/client/scripts/controllers/server/profiles.ts [hunk 2]
```diff
@@ -121,7 +121,6 @@ class ProfilesController {
                 ? JSON.parse(profileData.data)
                 : {};
               const lastActive = profileData.Address.last_active;
-              const isCouncillor = profileData.Address.is_councillor;
               const isValidator = profileData.Address.is_validator;
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/controllers/server/profiles.ts:124]: We can kill isValidator too, I assume.

File: packages/commonwealth/client/scripts/controllers/chain/substrate/identity.ts [hunk 3]
```diff
@@ -210,26 +210,4 @@ export default class SubstrateIdentity
       `${this.account.address} cleared identity`
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/controllers/chain/substrate/identity.ts:210]: TBD whether we can kill identity too.

File: packages/commonwealth/client/scripts/views/pages/new_proposal/substrate_democracy_proposal_form.tsx [hunk 4]
```diff
@@ -30,23 +28,6 @@ export class SubstrateDemocracyProposalForm extends ClassComponent {
   view() {
     const author = app.user.activeAccount as SubstrateAccount;
     const substrate = app.chain as Substrate;
-
-    let dataLoaded;
-
-    if (!author.isCouncillor) {
-      dataLoaded = false;
-    } else {
-      dataLoaded = !!substrate.democracyProposals?.initialized;
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/views/pages/new_proposal/substrate_democracy_proposal_form.tsx:39]: is this safe to remove?

**Review by jnaviask — COMMENTED** [review]
Leaving some comments here. OK to rip out identity, though -- which will require some more removals.

**Comment by jnaviask** [comment]
@affordances feel free to rip out all substrate identity related code, e.g. display names that override their profile, etc. Feel free to flag server routes that impact it as well.

@timolegros feel free to rip out chain events code for substrate identities (and the above proposal categories) if you'd like.

**Comment by zakhap** [comment]
testing this on staging first

File: packages/commonwealth/client/scripts/controllers/server/profiles.ts (hunk 2 above)
Comment (CowMuon) [reply packages/commonwealth/client/scripts/controllers/server/profiles.ts:124]: Right, I thought we said we were going to do that when we discussed this last week.

**Comment by jnaviask** [comment]
http://localhost:8080/edgeware/new/proposal/democracyproposal is broken.

Otherwise, good to go.

**Comment by jnaviask** [comment]
Noting that this functionality is also broken on production, so this PR is not a regression. G2G

**Review by jnaviask — APPROVED** [review]

**Pull request merged**